frontend/scripts/react-components/profile-root/profile-root.thunks.js

```diff
@@ -0,0 +1,9 @@
+import { getTopProfiles } from 'react-components/profile-root/profile-root.actions';
+
+export const loadTopNodes = (dispatch, getState) => {
+  const state = getState();
+  if (state?.app?.profileRoot?.topProfiles) {
```

Comment:
I've stopped trying to do this, ends up in bugs more often than not. The browser already has a good enough cache